apps/front-end/front-office/src/app/components/quiz-view/game-question/game-question.component.ts

```diff
@@ -63,6 +76,46 @@ export class GameQuestionComponent implements OnDestroy, OnInit {
     this.image_enabled = event;
   }
 
+  onSelectAnswer(answer: Quiz.Answer, index: number) {
+    const question = document.querySelector('#answer-' + index);
+
+    if (!question || question.classList.contains('disabled')) {
+      return;
+    }
+
+    question.classList.add('selected');
+
+    this.store.dispatch(
+      GameActions.chooseAnswer({ isCorrect: answer.isCorrect })
+    );
+
+    if (!answer.isCorrect) {
+      this.handleAnswerError(question);
+    }
+
+    this.store.dispatch(GameActions.nextQuestion());
+  }
+
+  handleAnswerError(question: Element) {
+    this.maxTries--;
+
+    if (this.maxTries <= 0) {
```

Comment:
should be using accomodation

Reply:
Accommodation set maxTries to 2 instead of 1